Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py

```diff
@@ -1334,9 +1325,8 @@ def get_domains_for_nameserver_command():
     # user input validation
     if not isinstance(nameservers, list):
         return_error('Name Servers list is not formatted correctly, please try again.')
-    if sort:
-        if sort != 'created' and sort != 'updated':
-            return_error('The parameter sort accept only these values: created/updated')
+    if sort and sort != 'created' and sort != 'updated':
+        return_error('The parameter sort accept only these values: created/updated')
```

Comment:
```suggestion
    if sort and sort not in ['created', 'updated']:
        return_error('The parameter sort accept only these values: created/updated/expired.')
```
and let's throw an error and catch from the main()